src/main/java/core/basesyntax/dao/fileReaderImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.dao;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class fileReaderImpl implements fileReader {
```

Comment:
The implemented interface `fileReader` should be renamed to `FileReader` to follow Java naming conventions.